aim/aim_manager.py

```diff
@@ -177,6 +180,8 @@ def create(self, context, resource, overwrite=False, fix_ownership=False):
                     context.store.from_attr(old_db_obj, type(resource),
                                             attr_val)
             db_obj = old_db_obj or context.store.make_db_obj(resource)
+            if 'name' in dir(db_obj):
```

Comment:
Isn't this done already in the resource's constructor object (which is needed in order to create the entry in the DB)?

Reply:
This change was added by @sayalinaval . I think it would be better if she explains this change.

Reply:
This can be removed.
And also line 245-246,273-274 should be removed from aim/tests/unit/test_aim_manager.py

Reply:
Thank you @sayalinaval , will do